packages/o/open-iscsi/pspec_x86_64.xml

```diff
@@ -0,0 +1,142 @@
+<PISI>
+    <Source>
+        <Name>open-iscsi</Name>
+        <Homepage>https://www.open-iscsi.com/</Homepage>
+        <Packager>
+            <Name>Jakob Gezelius</Name>
+            <Email>jakob@knugen.nu</Email>
+        </Packager>
+        <License>GPL-2.0-or-later</License>
+        <PartOf>network.utils</PartOf>
+        <Summary xml:lang="en">The Open-iSCSI project provides a high-performance, transport independent, implementation of RFC 3720 iSCSI for Linux.</Summary>
+        <Description xml:lang="en">The Open-iSCSI project provides a high-performance, transport independent, implementation of RFC 3720 iSCSI for Linux.
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>open-iscsi</Name>
+        <Summary xml:lang="en">The Open-iSCSI project provides a high-performance, transport independent, implementation of RFC 3720 iSCSI for Linux.</Summary>
+        <Description xml:lang="en">The Open-iSCSI project provides a high-performance, transport independent, implementation of RFC 3720 iSCSI for Linux.
+</Description>
+        <PartOf>network.utils</PartOf>
+        <Files>
+            <Path fileType="config">/etc/iscsi/initiatorname.iscsi</Path>
+            <Path fileType="config">/etc/iscsi/iscsid.conf</Path>
+            <Path fileType="config">/etc/logrotate.d/iscsiuiolog</Path>
+            <Path fileType="config">/etc/udev/rules.d/50-iscsi-firmware-login.rules</Path>
```

Comment:
It still ships files in `/etc`, we don't want that. Especially with migration to serpent tooling which can only write file to `/usr`. These files should be moved to `/usr` and `open-iscsi` needs to be teached to read it from that directory, which means it needs to patched. You can file an issue upstream, for them to work on it or you do it yourself. Either way it is probably not going be easy...